### PR TITLE
Amends

### PR DESCRIPTION
**Pull request by muzafarova**
Closes https://github.com/universalbasket/css/issues/32

- Removes shadow style from buttons and inputs and drops `.button--flat` class 
- Adds `option { color: #000 }` fix for Win
- Adjusts badges to be visually distinguishable from buttons. Scales down (.badge is ex .badge--small, .badge--large is ex .badge). .badge--inline is for usage in tables 
- Adjusts buttons and inputs to fit into `--control-height`, `--control-height--small` and ``--control-height--large` grid
- Some minor fixes 
- Updates demo page

### CODE REVIEW

File: stylesheets/components/input.css [hunk 1]
```diff
@@ -7,18 +7,17 @@
...
     border-radius: var(--border-radius);
     border: 1px solid var(--border-color);
-    box-shadow: 0 1px 2px rgba(2, 36, 102, .15) inset;
     background-color: var(--input-background);
     color: var(--input-color);
 }
 
 .input--small {
-    font-size: var(--font-size--small);
+    font-size: .95em;
```
Comment (inca) [inline stylesheets/components/input.css]: Interesting... 🤔 Any (new) guidelines around using ems?

File: stylesheets/helpers/utils.css [hunk 2]
```diff
@@ -87,10 +103,43 @@
...
 .stretch {
     width: 100%;
     justify-items: stretch;
 }
 
+/* POSITIONING */
+
+.sticky {
+    position: sticky;
+    z-index: 2;
```
Comment (inca) [inline stylesheets/helpers/utils.css]: Imo z-scale should be either standardized or left outside the core (no objections to keep it as it is though)

File: stylesheets/components/input.css (hunk 1 above)
Comment (muzafarova) [reply stylesheets/components/input.css]: this one comes from visual perception: default small size of 0.8em looks too small in the input box. 

The visual difference between regular and small input comes mostly from decreased paddings. 
I couldn't find any actual UBIO designs that use smaller inputs so you can look at this one as a draft. Feel free to change/request a change as soon as it becomes clear that small input should look different. Our base font size (12px) is quite small already, and it's not a common practice to use smaller font sizes in inputs on the web (it's usually something ok-readable as 12px on desktop or 16px on mobile)

File: stylesheets/helpers/utils.css (hunk 2 above)
Comment (muzafarova) [reply stylesheets/helpers/utils.css]: agree, I'll remove this one

**Pull request merged**